presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -6163,6 +6163,93 @@ public void testCorrelatedScalarSubqueries()
                 UNSUPPORTED_CORRELATED_SUBQUERY_ERROR_MSG);
     }
 
+    @Test
+    public void testCorrelatedNonAggregationScalarSubqueries()
+    {
+        String subqueryReturnedTooManyRows = "Scalar sub-query has returned multiple rows";
+
+        // multiple subquery output projections
+        assertQueryFails(
+                "SELECT name FROM nation n WHERE 'AFRICA' = (SELECT 'bleh' FROM region WHERE regionkey > n.regionkey)",
```

Comment:
@kokosing Grzegorz, I can't figure out why this query reports a different error then the next one. Would you explain? I'd expect this one to fail at runtime, not at planning time.